### DISCUSSION TITLE
Should hyper v1's documentation mention how to use it with Tokio?

### DISCUSSION THREAD

**Discussion by vi** [hyperium/hyper#3408] [General]
Hyper v1, unlike v0.14, is less tied to Tokio and requires [a special `unsafe`-using adaptor](https://github.com/hyperium/hyper/blob/master/benches/support/tokiort.rs) to make Hyper v1 speak in Tokio's terms.

This adapter is even used in [a client example](https://github.com/hyperium/hyper/blob/master/examples/client.rs), which is linked [from the docs](https://docs.rs/hyper/latest/hyper/client/index.html), making it trickier to use the example as a template for a project.

Should Hyper's docs mention Tokio explicitly and hint users e.g. to use some crate which implements that `tokiort`?

**Comment by seanmonstar** [comment]
Yes, there is a helper in [hyper-util](https://github.com/hyperium/hyper-util/blob/master/src/rt/tokio_io.rs). I would like a guide at hyper.rs/guides to mention it.

**Reply by vi** [reply]
When some version of `hyper-util` is expected to be [released](https://crates.io/crates/hyper-util) to crates.io?

Update: it is now released.
